drivers/mei/pkg.yaml

```diff
@@ -0,0 +1,34 @@
+name: thunderbolt
```

Comment:
intended?

Reply:
No, thanks. I let myself get rushed this morning.